rockcraft.yaml

```diff
@@ -0,0 +1,119 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+---
+name: kafka  # the name of your ROCK
+base: ubuntu@22.04  # the base environment for this ROCK
+version: '3.6.1'  # just for humans. Semantic versioning is recommended
+summary: Apache Kafka OCI Image  # 79 char long summary
+description: |
+  Apache Kafka Image from Canonical, based
+  on [Ubuntu](https://ubuntu.com/). Receives security updates for Apache
+  Kafka and Ubuntu. This repository is free to use and exempted from
+  per-user rate limits.
+
+  Apache Kafka is an open-source distributed event streaming platform for
+  high-performance data pipelines, streaming analytics and integrations.
+  Read more on the [Apache Kafka website](https://kafka.apache.org/).
+  This image requires the
+  [Apache Zookeeper image](https://hub.docker.com/r/ubuntu/zookeeper)
+  to run properly.
+license: Apache-2.0  # your application's SPDX license
+
+platforms:  # The platforms this ROCK should be built on and run on
+  amd64:
+
+environment:
+  JAVA_HOME: /usr/lib/jvm/java-18-openjdk-amd64
+  ZOOKEEPER_HOST: "zookeeper"
```

Comment:
**question:** is this just a placeholder?

Reply:
Not really. I mean, the pebble Kafka service needs these (see command in line 37). If I were to put that on the environment of Pebble, that would not make it possible to override it. Overriding this works now just fine, which I also test [here](https://github.com/canonical/kafka-rock/pull/2/files#diff-2465a5fd98a33922a6194b3483e771e3ab22469b6fcd9ba4ead7007923361475R64).

The default is chosen such that in a simple docker deployment (where in the docker network is also deployed a zookeeper named `zookeeper` which is the name of the ubuntu zookeeper slim image) this works out of the box.